dwave/system/composites/cutoffcomposite.py

```diff
@@ -27,33 +27,50 @@
 
 
 class CutOffComposite(dimod.ComposedSampler):
-    """Composite to cut off small interactions.
+    """Composite to remove interactions below a specified cutoff value.
 
-    Removes interactions smaller than a given cutoff. Isolated
-    variables (after the cutoff) are also removed.
-
-    Note that if the problem had isolated variables before the cutoff, they
-    will also be affected.
+    Downsizes the binary quadratic model (BQM) submitted to the child sampler by retaining
+    only interactions with values commensurate with the sampler's precision. Also removes
+    variables isolated post- or pre-removal of these interactions from the BQM passed on to
+    the child sampler, setting these variables to values that minimize the original BQM's
+    energy for the returned samples.
 
     Args:
        sampler (:obj:`dimod.Sampler`):
-            A dimod sampler
+            A dimod sampler.
 
-        cutoff (number):
-            The lower bound for interaction bias magnitudes. Interactions
-            with biases less than cutoff are removed. Isolated variables
-            are also not sent to the child sampler.
+       cutoff (number):
+            Lower bound for absolute value of interactions. Interactions
+            with absolute values lower than `cutoff` are removed. Isolated variables
+            are also not passed on to the child sampler.
 
-        cutoff_vartype (:class:`.Vartype`/str/set, default='SPIN'):
-            Variable space to do the cutoff in. Accepted input values:
+       cutoff_vartype (:class:`.Vartype`/str/set, default='SPIN'):
+            Variable space to execute the removal in. Accepted input values:
 
             * :class:`.Vartype.SPIN`, ``'SPIN'``, ``{-1, 1}``
             * :class:`.Vartype.BINARY`, ``'BINARY'``, ``{0, 1}``
 
-        comparison (function, optional):
-            A comparison operator for comparing the bias magnitude to the cutoff
+       comparison (function, optional):
+            A comparison operator for comparing interaction values to the cutoff
             value. Defaults to :func:`operator.lt`.
 
+    Examples:
+        This example removes one interaction, `'ac': -0.7`, before embedding
+        on a D-Wave system. Note that the lowest-energy sample for the embedded problem
+        is `{'a': 1, 'b': -1, 'c': -1}` but with a large enough number of samples
+        (here `num_reads=1000`), the lowest-energy solution to the complete BQM is
+        likely found and its energy recalculated by the composite.
+
+        >>> import dimod
+        >>> sampler = DWaveSampler(solver={'qpu': True})    # doctest: +SKIP
```

Comment:
don't need to skip this step

Reply:
It won't return an error without an API token?

Reply:
The doctests run with a token

Reply:
Removed